src/vivarium_cluster_tools/psimulate/worker/vivarium_work_horse.py

```diff
@@ -131,7 +138,7 @@ def work_horse(job_parameters: dict) -> Tuple[pd.DataFrame, Dict[str, pd.DataFra
         logger.info(f"Exiting job: {job_parameters}")
 
 
-def setup_sim(job_parameters: JobParameters) -> ParallelSimulationContext:
+def add_manager_configurations(job_parameters: JobParameters) -> JobParameters:
```

Comment:
What does this have to do with managers? Isn't it about branch configurations?

Reply:
the thing it is adding to the `branch_configuration` is the configuration for the artifact and randomness managers. It seems like putting them in "branch_configuration" in the first place is kind of a misleading choice. Steve and I felt like this name conveys that intent better